xpcPlugin/DataManager.cpp

```diff
@@ -792,21 +793,61 @@ namespace XPC
 		Set(DREF_FlapActual, value);
 	}
 
-	void DataManager::Execute(const std::string& comm)
+	void DataManager::MomentaryCommand(const std::string& comm)
```

Comment:
I prefer that function names be verbs. In this case, I think `ExecuteCommand` is fairly clear, although `ExecuteMomentaryCommand` could be okay (it's just a bit long).

Reply:
I think ExecuteMomentaryCommand is more clear - made the change.